run/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+
+if [ -z "$*" ]; then
+  echo "Expecting one or more arguments"
+  exit 1
+fi
+# shellcheck source=scripts/spack_setup.sh
+source "$GITHUB_WORKSPACE"/scripts/spack_setup.sh fetch
+spack install "$SPACK_SPEC"
+PATH="$PATH:$(spack location --install-dir "$SPACK_SPEC")/bin/"
+export PATH
+
+"$@"
```

Comment:
here we can do:

```bash
spack "$@"
```

so that actions do something like:

```hcl
action "install with spack" {
  uses = "popperized/spack@master"
  args = "install -f spack/spec.yml"
}
```

or execute any other arbitrary Spack command